### ISSUE TITLE
Lack info returned

### ISSUE THREAD

**Issue by general03** [danieliu/play-scraper#35]
**Is your feature request related to a problem? Please describe.**
I don't see the code where you extract `current_version`, `size` and `updated` in https://github.com/danieliu/play-scraper/blob/master/play_scraper/utils.py
Whereas in your README you explained that it's possible !

**Describe the solution you'd like**
I would like `details` method return this informations

Thanks

**Comment by danieliu** [comment]
The additional information section is parsed [here](https://github.com/danieliu/play-scraper/blob/d583ae866e9e379675d724ea1f859e51535c8365/play_scraper/utils.py#L168-L212). If these fields are being returned as None, it's likely because you're scraping in a different language as the mapping is hardcoded in english.

Please create a new issue if you're seeing unexpected behavior.

**Comment by general03** [comment]
Ok, I found the problem
```
    additional_info_data = parse_additional_info(
        soup.select_one('.xyOfqd'))
```
The class xyOfqd depend on language ? In the play store webpage I found IxB2fe class instead of xyOfqd

When I update this class name all works

**Comment by danieliu** [comment]
This has been fixed in d583ae866e9e379675d724ea1f859e51535c8365. Please update to the latest version.
